plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -170,6 +170,17 @@ export const Modal = ({ toggleModal }: SearchModalProps) => {
   );
 };
 
+const Context = ({ children }: PropsWithChildren<{}>) => {
```

Comment:
maybe its possible to use the [`useSearchContextCheck`](https://github.com/backstage/backstage/blob/master/plugins/search-react/src/context/SearchContext.tsx#L87) hook here? 

Option 1:

```tsx
const Context = ({ children }: PropsWithChildren<{}>) => {
    const hasSearchContext = useSearchContextCheck()
    if(!hasSearchContext){
       return <>{children}</>;
    }
    
    return <SearchContextProvider>{children}</SearchContextProvider>;
}
```

Reply:
Great idea, I will do it!